Sources/main.swift

```diff
@@ -0,0 +1,75 @@
+import HTTPServer
+import Router
+import JSON
+
+func getIdParameter(from request: Request) -> Int {
```

Comment:
If you have to do something like this, it means middleware has failed you :stuck_out_tongue:. I'll show how to replace this further down.